api/src/main/java/com/microsoft/gctoolkit/message/DataSourceParser.java

```diff
@@ -1,9 +1,13 @@
 package com.microsoft.gctoolkit.message;
 
+import com.microsoft.gctoolkit.aggregator.EventSource;
 import com.microsoft.gctoolkit.jvm.Diary;
 
+import java.util.Set;
+
 public interface DataSourceParser extends DataSourceChannelListener {
     void publishTo(JVMEventChannel channel);
     void diary(Diary diary);
     boolean accepts(Diary diary);
+    Set<EventSource> producesEvents();
```

Comment:
Grammatically, `eventsProduced` might be better

Reply:
'A DataSourceParser produces events of these types,' versus 'The events produced by a DataSourceParser are of these types'. IDK. I still lean toward producesEvents. 

But then... dataSourceParsers.stream().map(DataSourceParser::producesEvents) versus dataSourceParsers.stream().map(DataSourceParser::eventsProduced) makes me lean more toward eventsProduced. 

@karianna @kirk-microsoft Please weigh in.

Reply:
Kirk and I discussed and were astonished that such a great suggestion would come from @johnoliver.